lib/main.dart

```diff
@@ -23,6 +24,7 @@ void main() async {
     ));
     await MainBindings().dependencies();
     await HiveCacheConfig().setUp();
+    await Executor().warmUp();
```

Comment:
we need to check web or mobile to warmUp?

Reply:
checked